src/routes/properties/propertiesRoute.ts

```diff
@@ -50,6 +60,11 @@ function toResponse(property: Property, toUrl: (key: string) => string): Propert
     landmarks: property.landmarks,
     opportunities: property.opportunities,
     price: property.price,
+    ratings: property.ratings,
```

Comment:
why do we need individual ratings in API response?

Reply:
if we don't, lets remove them

Reply:
ok